Dockerfile

```diff
@@ -15,8 +17,5 @@ COPY package.json yarn.lock ./
 RUN yarn install
```

Comment:
In that case I think we only need to install the runtime dependencies, so we can probably add in `--production`

Reply:
The added `ENV NODE_ENV=production` above covers this case.

Reply:
cool 👍